card-style.cls

```diff
@@ -36,6 +37,27 @@
 
 \newcounter{cards}
 
+\newcommand{\CardType}[1]{%
+  \def\cardcolor{}
+  \ifnum\pdfstrcmp{#1}{Item}=0      \def\cardcolor{blue} \fi
+  \ifnum\pdfstrcmp{#1}{Action}=0    \def\cardcolor{yellow} \fi
+  \ifnum\pdfstrcmp{#1}{Spell}=0     \def\cardcolor{green} \fi
+  \ifnum\pdfstrcmp{#1}{Cantrip}=0   \def\cardcolor{green} \fi
+  \ifnum\pdfstrcmp{#1}{Focus}=0     \def\cardcolor{green} \fi
+  
+  \ifnum\pdfstrcmp{\cardcolor}{}=0 \else
+    \tikz[remember picture,overlay] {%
+      \draw [color=\cardcolor,line width=2mm]
+        (current page.south west)
+        rectangle
+        (current page.north east);
+    }
+  \fi
+}
+
+\xdef\CurrentCardCategory{None}
+\AddToHook {shipout/background}{\CardType{\CurrentCardCategory}}
```

Comment:
I'm confused about the words "category" vs "type" here.

Reply:
Indeed, we should use more consistent phrasing.